### PR TITLE
Day23

### PR DESCRIPTION
**Pull request by hoangcao16**

### CODE REVIEW

File: Day23/HoangNC/src/App.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import "./App.css";
+import ShowCurrent from "./components/Weather";
+import Footer from "./components/Footer";
+
+function App() {
+  return (
+    <div className="App">
+      <ShowCurrent></ShowCurrent>
```
Comment (tuyenbq-edsolabs) [inline Day23/HoangNC/src/App.js]: self-closed

Comment (hoangcao16) [reply Day23/HoangNC/src/App.js]: Hì e fix rồi ah

**Comment by hoangcao16** [comment]
Capture

**Comment by hoangcao16** [comment]
DONE

**Pull request merged**